msbuild/tests/Xamarin.iOS.Tasks.Tests/ProjectsTests/CodesignAppBundle.cs

```diff
@@ -13,8 +13,8 @@ namespace Xamarin.iOS.Tasks
 {
 	[TestFixture ("iPhone", "Debug")]
 	[TestFixture ("iPhone", "Release")]
-	[TestFixture ("iPhoneSimulator", "Debug")]
-	[TestFixture ("iPhoneSimulator", "Release")]
+	//[TestFixture ("iPhoneSimulator", "Debug")]
```

Comment:
Why are these commented out?

Reply:
They fail because they are no longer accurate tests - each appex/watch app is signed/not-signed based on its own inputs, not the inputs of the top-level project.

Reply:
Please comment in line if that is true

// TestFixture (Blah) - This case is not valid because Foo

so future us knows why they are commented out.

Reply:
So if I understand correctly the device variations work because we always sign everything on device, but for the simulator we only sign sometimes.

Could you add a comment explaining that? As it stands it looks like a FIXME :smile: